examples/sparse/c_and_s.py

```diff
@@ -0,0 +1,143 @@
+"""
+[Combining Label Propagation and Simple Models Out-performs
+Graph Neural Networks](https://arxiv.org/abs/2010.13993)
+"""
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from dgl.data import CoraGraphDataset
+from dgl.mock_sparse import create_from_coo, diag, identity
+from torch.optim import Adam
+
+
+class LabelPropagation(nn.Module):
+    def __init__(self, num_layers=20, alpha=0.9):
+        super().__init__()
+
+        self.num_layers = num_layers
+        self.alpha = alpha
+
+    ###########################################################################
+    # (HIGHLIGHT) Compute Label Propagation with Sparse Matrix API
+    ###########################################################################
+    @torch.no_grad()
```

Comment:
Shall we make this a preprocessing step like SIGN?

Reply:
That's definitely feasible. A follow-up question is weather we still want to have a `CorrectAndSmooth` class or simply two functions `correct` and `smooth`.